packages/inertia-react/src/App.js

```diff
@@ -18,12 +18,7 @@ export default function App({
   })
 
   const headManager = useMemo(() => {
-    const isServer = typeof window === 'undefined'
-    const headManager = createHeadManager(isServer)
-    if (isServer && onHeadUpdate) {
-      headManager.onUpdate(onHeadUpdate)
-    }
-    return headManager
+    return createHeadManager(typeof window === 'undefined', onHeadUpdate || (() => {}))
```

Comment:
Maybe we can split this into multiple lines, so that it's easier to visually see that `x || y` second argument?

Reply:
Hah, I had it that way, and then switched it back. But yeah, you're probably right. I'll make that change elsewhere.